src/QsCompiler/Tests.Compiler/EntryPointSchemaTests.fs

```diff
@@ -77,10 +79,10 @@ type EntryPointSchemaTests(output: ITestOutputHelper) =
     let sampleEntryPointOperations =
         Map
             .empty
-            .Add("UseNoArgs", (new EntryPointOperation(Name = "UseNoArgs"), "{\"Name\":\"UseNoArgs\"}"))
+            .Add("UseNoArgs", (new EntryPointOperation(Name = "UseNoArgs"), Path.Join(testCasesDirectory, "UseNoArgs.json") |> File.ReadAllText))
```

Comment:
Instead of the value of this map being a tuple, it can just be the `EntryPointOperation` object and the test functions can read the text from the file using the key as the file name (plus the .json extension) in a known directory.

This way the `Path.Join` and `File.ReadAllText` operations are only performed in one place.